src/components/Navbar/BurgerMenu.js

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import { slide as Menu } from 'react-burger-menu';
+export default function BurgerMenu (props) {
+ console.log(props)
+var styles = {
+  bmBurgerButton: {
+    position: 'fixed',
+    width: '36px',
+    height: '30px',
+    left: props.burMenuButtonPosition,
+    top: '10px'
+  },
+  bmBurgerBars: {
+    background: '#373a47'
+  },
+  bmBurgerBarsHover: {
+    background: '#a90000'
+  },
+  bmCrossButton: {
+    height: '24px',
+    width: '24px'
+  },
+  bmCross: {
+    background: '#bdc3c7'
+  },
+  bmMenuWrap: {
+    position: 'fixed',
+      height: '100%',
+    width: 'fit-content',  
+  },
+  bmMenu: {
+    background: '#FFFFFF',
+    padding: '2.5em 1.5em 0',
+    width: '25vw',
+      minWidth: '140px',
+      maxWidth: '300px',
+    fontSize: '1.15em',
+      overflow: 'hidden',
```

Comment:
this will lead to a bug; we need overflow-y atleast

Reply:
Can you test and reproduce the bug?

Reply:
@BBoutet1  reduce height of ur page less than the height of sidebar-menu and u will also see the issue that i saw
![image](https://user-images.githubusercontent.com/6857593/95688670-6ef9f700-0c29-11eb-9546-cfa2dff2c6d6.png)

Reply:
@harshvchawla Ok. Will fix that